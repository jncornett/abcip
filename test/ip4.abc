d ( stack="eth:ip4:udp" )
a ( ip4:opt="|0 0 0 0 0 0 0 0|"; len=16 )
a ( ip4:opt="|88 4 a 5|"; len=16 )
a ( ip4:opt="|0 0 0 0|"; len=16 )
