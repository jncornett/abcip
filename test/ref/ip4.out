IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto UDP (17), length 52, options (EOL))
    10.1.2.3.48620 > 10.9.8.7.8: UDP, length 16
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4700  ..............G.
	0x0010:  0034 0001 0000 4011 5aa5 0a01 0203 0a09  .4....@.Z.......
	0x0020:  0807 0000 0000 0000 0000 bdec 0008 0018  ................
	0x0030:  dcdb 4c6f 7265 6d20 6970 7375 6d20 646f  ..Lorem.ipsum.do
	0x0040:  6c6f                                     lo
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto UDP (17), length 48, options (unknown 136))
    10.1.2.3.48620 > 10.9.8.7.8: UDP, length 16
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4600  ..............F.
	0x0010:  0030 0002 0000 4011 c99e 0a01 0203 0a09  .0....@.........
	0x0020:  0807 8804 0a05 bdec 0008 0018 0527 7220  .............'r.
	0x0030:  7369 7420 616d 6574 2c20 636f 6e73       sit.amet,.cons
IP (tos 0x0, ttl 64, id 3, offset 0, flags [none], proto UDP (17), length 48, options (EOL))
    10.1.2.3.48620 > 10.9.8.7.8: UDP, length 16
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4600  ..............F.
	0x0010:  0030 0003 0000 4011 5ba7 0a01 0203 0a09  .0....@.[.......
	0x0020:  0807 0000 0000 bdec 0008 0018 cba5 6563  ..............ec
	0x0030:  7465 7475 7220 6164 6970 6973 6369       tetur.adipisci
